drivers.json

```diff
@@ -28,6 +28,26 @@
       "Hardware-enforced Stack Protection Compatibility Test"
     ]
   },
+  "igb": {
+    "name": "Intel(R) 82576 Gigabit Dual Port Network Connection",
+    "device": "igb",
+    "type": 0,
+    "install_method": "no-drv",
+    "support": true,
+    "pretestcommands": [
+      {
+        "desc": "Rename igb ethernet adapter to SupportDevice0",
+        "run": "Rename-NetAdapter -Name (Get-NetAdapter -InterfaceDescription 'Intel(R) 82576 Gigabit Dual Port Network Connection').Name -NewName 'SupportDevice0'"
+      }
+    ],
+    "ignore_list": [
+      "NDISTest 6.5 - [2 Machine] - GlitchFreeDevice",
+      "NDISTest 6.5 - [2 Machine] - VMQReceiveQueueStateChecking",
+      "NDISTest 6.5 - [2 Machine] - VMQPowerManagement",
+      "NDISTest 6.0 - [1 Machine] - 1c_Mini6PerfSend",
```

Comment:
1c_Mini6PerfSend - why is this test blacklisted? In general it should pass.

Reply:
The ignored tests pass when ran individually but fail in an automated test run.

Reply:
I am wondering what are the failures for _GlitchFreeDevice_ and _1c_Mini6PerfSend_ tests.

Reply:
For GlitchFreeDevice test case 1083663865 "Make sure we received the expected number of packets at the adapter level " is failing:
```
Test Adapter Intel(R) 82576 Gigabit Dual Port Network Connection received 3905 packets and dropped 0 packets out of 12385 sent (based on ifHCInUcastPkts)
88888 Test adapter received 31% of packets, required 40% to pass. 
```

For 1c_Mini6PerfSend, variation 114 "Send results" is failing:
```
Variation #114 Send results
[GetSendResults](c:\hlk\JobsWorkingDir\Tasks\WTTJobRun12BC46A1-5B98-ED11-8ACC-56000600DDDD\ndistest\bin\..\commands.htm#GetSendResults)
  - EndPointID = 0x01020001.01
  - Comm Manager Type = 2
  - Constructor Type = 0
Performance Communication Manager Send Statistics:
  - Net Buffer Lists Sent            : 12882915
  - Calls To Send                    : 2576574
  - Net Buffer Lists Send Completed  : 12882915
  - Net Buffer Lists Sent Successfully: 3980
  - Total Counts                     : 1403193741
  - Start Count                      : 1243325246337
  - Counter Precision                : 100000000
  - Stop Send Counts                 : 1244724572765
  - Number of Processors             : 4
  - Number Of NBLs Time Stamped      : 0
  - Total Time Stamp Difference      : 0
Total Counts : 1403193741
Counter Precision: 100000000
FAILED: [88888] Sends failed on open 1
Processor Times & CPU Utilization: 
[0] Kernel Time        : 55937500
[0] Processor Time     : 140312500
[0] CPU Utilization    : 39.8%
[1] Kernel Time        : 57187500
[1] Processor Time     : 140312500
[1] CPU Utilization    : 40.7%
[2] Kernel Time        : 140156250
[2] Processor Time     : 140312500
[2] CPU Utilization    : 99.8%
[3] Kernel Time        : 46718750
[3] Processor Time     : 140000000
[3] CPU Utilization    : 33.3%
Total Kernel Time       : 300000000
Total Processor Time    : 560937500
Average CPU Utilization : 53.4
Cpu Utilization: 53.4
[GetSendResults](c:\hlk\JobsWorkingDir\Tasks\WTTJobRun12BC46A1-5B98-ED11-8ACC-56000600DDDD\ndistest\bin\..\commands.htm#GetSendResults)
  - EndPointID = 0x01020002.01
  - Comm Manager Type = 2
  - Constructor Type = 0
Performance Communication Manager Send Statistics:
  - Net Buffer Lists Sent            : 13152870
  - Calls To Send                    : 2630565
  - Net Buffer Lists Send Completed  : 13152870
  - Net Buffer Lists Sent Successfully: 7010
  - Total Counts                     : 2397848168
  - Start Count                      : 1243325245642
  - Counter Precision                : 100000000
  - Stop Send Counts                 : 1244724573465
  - Number of Processors             : 4
  - Number Of NBLs Time Stamped      : 0
  - Total Time Stamp Difference      : 0
Total Counts : 2397848168
Counter Precision: 100000000
FAILED: [88888] Sends failed on open 2
Processor Times & CPU Utilization: 
[0] Kernel Time        : 62187500
[0] Processor Time     : 239843750
[0] CPU Utilization    : 25.9%
[1] Kernel Time        : 59843750
[1] Processor Time     : 239843750
[1] CPU Utilization    : 24.9%
[2] Kernel Time        : 140937500
[2] Processor Time     : 239843750
[2] CPU Utilization    : 58.7%
[3] Kernel Time        : 48281250
[3] Processor Time     : 239531250
[3] CPU Utilization    : 20.1%
Total Kernel Time       : 311250000
Total Processor Time    : 959062500
Average CPU Utilization : 32.4
Cpu Utilization: 32.4
[GetSendResults](c:\hlk\JobsWorkingDir\Tasks\WTTJobRun12BC46A1-5B98-ED11-8ACC-56000600DDDD\ndistest\bin\..\commands.htm#GetSendResults)
  - EndPointID = 0x01020003.01
  - Comm Manager Type = 2
  - Constructor Type = 0
Performance Communication Manager Send Statistics:
  - Net Buffer Lists Sent            : 13240925
  - Calls To Send                    : 2648176
  - Net Buffer Lists Send Completed  : 13240925
  - Net Buffer Lists Sent Successfully: 6820
  - Total Counts                     : 2397829935
  - Start Count                      : 1243325244982
  - Counter Precision                : 100000000
  - Stop Send Counts                 : 1244724574322
  - Number of Processors             : 4
  - Number Of NBLs Time Stamped      : 0
  - Total Time Stamp Difference      : 0
Total Counts : 2397829935
Counter Precision: 100000000
FAILED: [88888] Sends failed on open 3
Processor Times & CPU Utilization: 
[0] Kernel Time        : 62187500
[0] Processor Time     : 239843750
[0] CPU Utilization    : 25.9%
[1] Kernel Time        : 59843750
[1] Processor Time     : 239843750
[1] CPU Utilization    : 24.9%
[2] Kernel Time        : 140937500
[2] Processor Time     : 239843750
[2] CPU Utilization    : 58.7%
[3] Kernel Time        : 48281250
[3] Processor Time     : 239843750
[3] CPU Utilization    : 20.1%
Total Kernel Time       : 311250000
Total Processor Time    : 959375000
Average CPU Utilization : 32.4
Cpu Utilization: 32.4
[GetSendResults](c:\hlk\JobsWorkingDir\Tasks\WTTJobRun12BC46A1-5B98-ED11-8ACC-56000600DDDD\ndistest\bin\..\commands.htm#GetSendResults)
  - EndPointID = 0x01020004.01
  - Comm Manager Type = 2
  - Constructor Type = 0
Performance Communication Manager Send Statistics:
  - Net Buffer Lists Sent            : 13057780
  - Calls To Send                    : 2611547
  - Net Buffer Lists Send Completed  : 13057780
  - Net Buffer Lists Sent Successfully: 6980
  - Total Counts                     : 2396256435
  - Start Count                      : 1243325243268
  - Counter Precision                : 100000000
  - Stop Send Counts                 : 1244724575116
  - Number of Processors             : 4
  - Number Of NBLs Time Stamped      : 0
  - Total Time Stamp Difference      : 0
Total Counts : 2396256435
Counter Precision: 100000000
FAILED: [88888] Sends failed on open 4
Processor Times & CPU Utilization: 
[0] Kernel Time        : 62187500
[0] Processor Time     : 239687500
[0] CPU Utilization    : 25.9%
[1] Kernel Time        : 59843750
[1] Processor Time     : 239687500
[1] CPU Utilization    : 24.9%
[2] Kernel Time        : 140937500
[2] Processor Time     : 239687500
[2] CPU Utilization    : 58.8%
[3] Kernel Time        : 48281250
[3] Processor Time     : 239687500
[3] CPU Utilization    : 20.1%
Total Kernel Time       : 311250000
Total Processor Time    : 958750000
Average CPU Utilization : 32.4
Cpu Utilization: 32.4

Variation Summary 

Total NBLs Sent : 52334490
Time Taken (seconds) : 23.97848168
NW Throughput (bps): 83810328060.7715
Overall CPU utilization: 53.4
```

Reply:
I would dig a little bit more into it. Best start with GlitchFreeDevice. We shouldn't have dropped packet in general. Could be small tap queue on host, but then we should be at least aware of it.
Sometime ago we enlarged tap queue during test.